### ISSUE TITLE
Popup keeps showing on a multi page app that uses page cache

### ISSUE THREAD

**Issue by Chris-Marassovich** [EdiWang/WhatsNew.UWP#2]
Hello,

Firstly great tool and thanks for building it.

Issue:
My issue is I have a main page where WhatsNew is implemented and a detail page.
In the users first session on first navigation to the main page the WhatsNew shows correctly.  User acknowledges the pop up.  The user navigates to detail page and on return to main page the WhatsNew pop up returns.  This will continue until the users closes this initial session and any subsequent session all is working as expected.

Solution:
As I look over your code (I am quite new to .NET so please excuse any ignorance on my part) I see that it expects to be instantiated on each page invocation.  It sets the Version Key but leaves the "state" at WhatsNewState.VersionUpdate.  In my case I keep my Main Page cached for technical reasons and this leads to the WhatsNew popup constantly showing on each navigation to the main page for the very first user session.

The contributor to this issue is my technical issue of having to keep my main page cached.  That I cannot change. 

I think the solution to this is to ensure the entire state of the WhatsNew objects is left in a correct state and not rely on being instantiated every time.
I think the fix is in the WhatsNewPopup class to set the state of the WhatsNewHelper to WhatsNewState.Active after the WhatsNew popup has been displayed.

In summary it appears that the control expects to be destroyed and instantiated between page invocations.  It does not set the entire state correctly and hence in a page invocation where the page is cached the WhatsNewPopup is left in the state WhatsNewState.VersionUpdate even though all versions numbers have been saved to storage.

I hope that is clear?  As I said I am quite new to .NET and I may not be correct in the solution I have detailed.  Certainly I get this issue because I cache my Main Page and hence the WhatsNewPopup stays cached.

Cheers
Chris ....

**Comment by EdiWang** [comment]
Thanks for your feedback. I will investigate the issue soon!
